### ISSUE TITLE
Docs: Show no documents by default

### ISSUE THREAD

**Issue by matteodem** [matteodem/meteor-easy-search#399]
see #396

**Comment by allenfuller** [comment]
+1. I'm trying to setup a Twitter-style search box and really bouncing around a lot of different documentation. If I come up with a solution, would it be helpful to post it here?

**Comment by matteodem** [comment]
Yes sure, right now there's also a bug that makes this harder https://github.com/matteodem/meteor-easy-search/issues/364.

**Comment by kartikTT** [comment]
Trying to accomplish the same, running into #364 as well. Any solutions/better approach to only showing results when searching?

**Comment by matteodem** [comment]
You can have some logic for now that checks the value of the input through the component dict. Should definitely be a parameter to pass in to the component, but I'm rather busy right now.

**Comment by kartikTT** [comment]
Thanks for the pointer @matteodem. I tried the following, where getComponentDict returns the searchDefinition:

                               `{{#if getComponentDict}}
					<h2>Results</h2>
						{{#EasySearch.Each index=index}}
							<div>{{name}}</div>
					       {{/EasySearch.Each}}
				{{/if}}`

The "Results" header shows up instantly when I type in the input, but the search results take >10 seconds to show up. If I remove the getComponentDict requirement, the searches show up ~ instantly.

Saw the same delay when I used EasySearch.IfInputEmpty as my if condition.

Any idea why there's a delay only for the search results?
